.github/workflows/maturin_ci.yml

```diff
@@ -3,14 +3,15 @@
 #
 #    maturin generate-ci github
 #
+# Prefer to run matrix of unit-tests in the pytest workflow until the following
+# issue is solved: https://github.com/PyO3/maturin/issues/1971
 name: Maturin CI
 
 on:
   push:
     paths:
       - '**/*.py'
       - '**/*.rs'
-      - 'pytest.ini'
```

Comment:
Do we want to run if the dependency files have changed (i.e. `requirements.txt` and `Cargo.lock`)?

Reply:
Good idea. I have also added `pyproject.toml` and `Cargo.toml` (it is easy to manually change those and forget about the generated ones)